pkgs/by-name/ce/celeste-classic/package.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenvNoCC, fetchzip, autoPatchelfHook, pkgs, lib, practiceMod ? false }:
+
+let
+  folder = if practiceMod then "CELESTE*Practice*" else "CELESTE";
+  srcbin = if practiceMod then "celeste_practice_mod" else "celeste";
+  outbin = if practiceMod then "celeste-classic-pm" else "celeste-classic";
+in stdenvNoCC.mkDerivation {
+  pname = outbin;
+  version = "1.0";
+
+  src = fetchzip {
+    url = "https://www.speedrun.com/static/resource/174ye.zip";
+    hash = "sha256-GANHqKB0N905QJOLaePKWkUuPl9UlL1iqvkMMvw/CC8=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = with pkgs; [ SDL2 ];
+
+  sourceRoot = ".";
```

Comment:
there's only a single top-level directory in the zip, so I don't think this is needed? 
```suggestion
```